backend/package.json

```diff
@@ -16,7 +16,8 @@
     "dotenv": "^16.5.0",
     "express": "^5.1.0",
     "openai": "^4.98.0",
-    "spotify-web-api-node": "^5.0.2"
+    "spotify-web-api-node": "^5.0.2",
+    "ts-node": "^10.9.2"
```

Comment:
I didn't know I had to install `ts-node` explicitly; I thought it just came with starting something in typescript.